### PR TITLE
[Documentation][ShopBundle][TheBook] twig money format helpers

### PR DESCRIPTION
**Pull request by Mipme**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | none |
| License         | MIT |

Handy Money Format helpers are already there in Sylius - but not
documented. Here u go! ;-)

### CODE REVIEW

File: docs/book/configuration/currencies.rst [hunk 1]
```diff
@@ -71,6 +71,22 @@ the ``Sylius\Component\Core\Currency\CurrencyStorageInterface`` with methods
 
     $container->get('sylius.storage.currency')->set($channel, 'PLN');
 
+Displaying Currencies in a templates
```
Comment (CoderMaggie) [inline docs/book/configuration/currencies.rst]: `...in the templates`

**Comment by lchrusciel** [comment]
@Mipme would you like to add the change proposed by @CoderMaggie?

File: docs/book/configuration/currencies.rst [hunk 2]
```diff
@@ -71,6 +71,22 @@ the ``Sylius\Component\Core\Currency\CurrencyStorageInterface`` with methods
 
     $container->get('sylius.storage.currency')->set($channel, 'PLN');
 
+Displaying Currencies in the templates
+------------------------------------
```
Comment (lchrusciel) [inline docs/book/configuration/currencies.rst]: Missing two last dashes

**Comment by CoderMaggie** [comment]
Thank you @Mipme ;) 🎉

**Pull request merged**